.github/actions/install-package/action.yml

```diff
@@ -30,7 +30,7 @@ runs:
 
       - name: Install dependencies
         run: |
-          pip install ".[dev,e2e]"
+          pip install -r dev_requirements.txt
```

Comment:
note: when merged we should update [notion env setup for client](https://www.notion.so/neptuneai/PyCharm-for-neptune-client-f8c8ba96d1c6469fa833f74dc302b2b7?pvs=4#6ff75edc9ed74b7e8d0aface7d20f612)

Reply:
Yes, it's noted on my TODO :wink: